test/helpers.bash

```diff
@@ -355,7 +355,8 @@ function run_in_host_netns() {
 # second arg is container_id
 # third is container name
 # fourth is subnet
-# fifth and greater are aliases
+# fifth is custom dns server for container (empty will not be used)
+# sixth and greater are aliases
 function create_config() {
```

Comment:
Nit, this is IMO not really maintainable. Adding arguments like will make it very hard to review and easy for callers to use it wrong. I would prefer an approach like this: https://github.com/containers/netavark/blob/d9e76ab68726cef40782d99f8e9df6e1d6d5c41f/test/helpers.bash#L325-L333
@edsantiago Do you have other suggestion to make this better?

Reply:
I'm fine to convert this to `named` arguments and parse it or something even better that @edsantiago suggests.

Reply:
@Luap99's suggestion looks great, and includes suitable error checking. I can't think of a better approach. (And, yes, I agree that positional args break down after about five).

Reply:
SGTM. I'll amend this test.